pivotal-import/lib.py

```diff
@@ -169,8 +169,19 @@ def populate_config():
         logger.debug(
             "Skipping populating config.json, because the file already exists."
         )
-        with open("config.json", "r") as f:
+        read_config_from_disk()
```

Comment:
- function param?
- is `return` needed?

Reply:
Thanks for catching this, pushed up a fix.